Entities/Items/Sponge/SpongeDryInAir.as

```diff
@@ -0,0 +1,26 @@
+#include "SpongeCommon.as";
+
+const int slow_frequency = 5; // don't run this script too often
+
+void onInit(CBlob@ this)
+{
+	this.getCurrentScript().tickFrequency = 30 * 2 * slow_frequency; // 0.5 "absorbs" per seconds, run only every 5 "absorbs"
```

Comment:
you can use the script flags tick_not_inwater and tick_ininventory (instead of doTickScripts).

Reply:
tick_ininventory does not seem to work for me. Is also not used anywhere...